addons/ofxOsc/libs/oscpack/src/ip/posix/UdpSocket.cpp

```diff
@@ -557,7 +557,7 @@ class SocketReceiveMultiplexer::Implementation{
 		break_ = true;
 
 		// Send a termination message to the asynchronous break pipe, so select() will return
-		write( breakPipe_[1], "!", 1 );
+		if(write( breakPipe_[1], "!", 1 )) {};
```

Comment:
Uh, this looks a bit odd ... `write()` should be able to stand alone, no? Does gcc print a warning for this line?

Reply:
Yes, unused return value.

Reply:
I see ... since this code is imported straight from [oscpack](https://github.com/mkalten/oscpack/blob/master/ip/posix/UdpSocket.cpp#L537) i would leave it as-is here, and send a pull request to oscpack. Otherwise, if oscpack ever gets updated, and we re-import the library, any changes might get lost.

Reply:
Good point. Cool you know the codebase so well.

I will send a PR to oscpack.